airbyte-config/config-models/src/main/resources/types/SyncStats.yaml

```diff
@@ -21,3 +21,7 @@ properties:
     type: integer
   recordsCommitted:
     type: integer # if unset, committed records could not be computed
+  meanSecondsBeforeStateMessageEmitted:
+    type: integer
+  maxSecondsBeforeStateMessageEmitted:
+    type: integer
```

Comment:
A comment is probably valuable for future us - that this message is from the Source.